visualization/app/codeCharta/ui/codeMap/codeMap.actions.service.ts

```diff
@@ -60,7 +60,7 @@ export class CodeMapActionsService {
 		for (const edge of edges) {
 			edge.visible = EdgeVisibility.none
 
-			if (edge.attributes[edgeMetric] !== undefined && edgePreviewNodes !== null) {
```

Comment:
We should check for a Set initialized with an empty [], if the else-branch in line 73 implements a valid behaviour.

Reply:
You mean for correctness? 
Because while we set visibility for to-edges they are empty as well, so no edges are drawn on the canvas.

Reply:
I guess one could argue that the current result is incorrect, because the function shows
both if both exist
to if its the only existing direction
from else

However in an empty case the last transformation would lead to empty, which is not an expected result according to such a definition

Reply:
I deleted my previous comments
The first if would catch an empty set because from == to == [] then the inner if checks it from is [] and only if it isn't it sets visibility

Reply:
Missed that point as well, but now we can be sure there is no issue

Reply:
Thanks, there is definitely no bug :)